hieradata/common.yaml

```diff
@@ -574,7 +574,7 @@ govuk_ci::master::pipeline_jobs:
   govuk-diff-pages: {}
   govuk_admin_template: {}
   govuk_ab_testing: {}
-  govuk_client_url-arbiter: {}
+  govuk-client-url_arbiter: {}
```

Comment:
The [url-arbiter app is dead](https://github.com/gds-attic/url-arbiter) and the client should be too - we should be able to delete this.